IPCloneDetectorAndASNAnalyzer/IPCloneDetectorAndASNAnalyzer.php

```diff
@@ -0,0 +1,58 @@
+<?php
+/**
+  @title IP Clones and ASN Analysis
+  @author Madrix
+  @description Displays some statistics about ASNs, IPs, and clones.
+  @contact 
+  @version 1.0
+  @tested 0.9
+  @minver 0.9
+  @maxver *
+  @license GPLv3
+  @icon https://github.com/unrealircd/unrealircd-webpanel-plugins/blob/main/example_plugin/screenshots/example_icon.png?raw=true
```

Comment:
Please could you use your own icons and screenshots instead of the ones from the example plugin